lib/aikido/zen/agent/heartbeats_manager.rb

```diff
@@ -30,11 +30,11 @@ def stale_settings?
     def start(&task)
       return if running?
 
-      if @settings.heartbeat_interval&.nonzero?
```

Comment:
I can't really follow this change, isn't it still called `heartbeat_interval` in `runtime_settings.rb`?

Reply:
I just missed this alias at the end of the file. :D

```rb
 def interval
      @settings.heartbeat_interval
    end
```